github/repos_autolinks.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// AutolinkOptions specifies parameters for RepositoriesService.AddAutolink method
+// Also this struct is embedded into an Autolink struct which contains one additional ID field
+type AutolinkOptions struct {
+	KeyPrefix   *string `json:"key_prefix,omitempty"`
+	URLTemplate *string `json:"url_template,omitempty"`
+}
+
+// Autolink embeds an AutolinkOptions struct and is used by
+// RepositoriesService.ListAutolinks, RepositoriesService.AddAutolink and RepositoriesService.GetAutolink methods
+type Autolink struct {
+	ID *int64 `json:"id,omitempty"`
+	*AutolinkOptions
+}
+
+// ListAutolinks returns a list of autolinks configured for the given repository.
+// Information about autolinks are only available to repository administrators.
+// GitHub API docs: https://docs.github.com/en/rest/reference/repos#list-all-autolinks-of-a-repository
```

Comment:
Please add a blank comment line immediately before this line to separate the GitHub API docs for consistency with the other methods in the repo.